### DISCUSSION TITLE
Custom fields filtered by object_types

### DISCUSSION THREAD

**Discussion by variable** [netbox-community/pynetbox#696] [General]
### pynetbox version

v7.0.1

### NetBox version

4.0.5

### Feature type

New functionality

### Proposed functionality

Is there a way to filter by object_types in custom fields? eg. 
![image](https://github.com/netbox-community/pynetbox/assets/558175/27720e0b-bc70-4c4b-a86d-51f85d8efb89)

I tried various ways with no success

### Use case

So I can filter with the name and the object_types to ensure the result is accurate instead of just filtering by the name

**Comment by jnovinger** [comment]
@variable , are you asking about filtering instances of object types that have custom fields attached that might represent another object type? Or are you asking about filtering custom field instances (e.g. custom field definitions)?
